fragments/prj.conf.ashell

```diff
@@ -1,14 +1,10 @@
-# HEAP
-CONFIG_HEAP_MEM_POOL_SIZE=4096
-
 # SHELL
 CONFIG_EARLY_CONSOLE=y
```

Comment:
if you are already get these configs from prj.conf.base, then there's a lot of duplicate here like STDOUT_CONSOLE, FLOAT, NEWLIB